collection-scripts/gather_admission_webhooks

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-BASE_COLLECTION_PATH="/must-gather"
-WEB_HOOK_PATH="${BASE_COLLECTION_PATH}/web_hooks/"
-
-mkdir -p ${WEB_HOOK_PATH}
-oc get mutatingwebhookconfigurations -o yaml > ${WEB_HOOK_PATH}/mutatingwebhookconfigurations.yaml
```

Comment:
This script/code collects all `mutatingwebhookconfigurations` and `validatingwebhookconfigurations` and places them in the `${BASE_COLLECTION_PATH}/web_hooks` directory. 

Thus your [comment](https://github.com/openshift/must-gather/pull/325#issue-1317892153) is accurate in that this information is duplicated; but only because the [kube-apiserver (cluster operator) is defining `.status.relatedObjects` ](https://github.com/openshift/enhancements/blob/master/enhancements/oc/inspect.md)which also collects this information.

Thus its probably best that we only remove the [comment](https://github.com/openshift/must-gather/pull/325/files#diff-bad6eb3f5953b7f47f014fd8b04a48470c1d64d94bbd29566af842f0b60c909cL73) about the kubeapi-server doing this (because it now does this). However the other collections (or duplications) may also be warranted (IMO) as we may want to know about what other  `mutatingwebhookconfigurations` and `validatingwebhookconfigurations` are in place. 

We want to do this for 2 reasons: 

1. we may not be collecting the 'resource' that defines the  `.status.relatedObjects` relationship; and thus would not be able to see / understand if something causing mutating or validation behaviors. 

1. for resources we collect, they may don't properly define this `.status.relatedObjects` relationship, and thus would not be able to see / understand if something causing mutating or validation behaviors, if such a hook was in place. 


Thus it might be prudent for us to keep this script in place.

Reply:
> we may not be collecting the 'resource' that defines the .status.relatedObjects relationship; and thus would not be able to see / understand if something causing mutating or validation behaviors.

This would be an operator bug, so a risk, but if it's not observed in the field, I'd allow this.

> 2\. for resources we collect, they may don't properly define this `.status.relatedObjects` relationship, and thus would not be able to see / understand if something causing mutating or validation behaviors, if such a hook was in place.

This would be an `oc adm inspect` bug, so a risk, but if it's not observed in the field, I'd allow this.


Question to the author: have you considered expanding `oc adm inspect` to add namespaces containing services references by these webhooks?

Reply:
> Question to the author: have you considered expanding oc adm inspect to add namespaces containing services references by these webhooks?

That may be useful when we have webhooks related issue. It could be considered an unexpected behavior for users if we gather user namespace info automatically as it may contain other apps and/or sensitive information. I'm not sure if we want this behavior with standard cluster operator must-gather.